IdentityManagement/Sessions/session.ts

```diff
@@ -0,0 +1,44 @@
+const express = require('express')
+const app = express()
+const port = 3000
+const axios = require('axios');
+const jwt = require("jsonwebtoken")
+
+const jwtKey = "my_secret_key"
```

Comment:
We should load this from an environment variable so that we don't need to commit the secret key to the public repo